core/src/main/java/lucee/runtime/engine/ThreadLocalPageContext.java

```diff
@@ -124,7 +132,14 @@ public static Config getConfig(Config config) {
 		return config;
 	}
 
+	public static boolean isPreciseMathGloballyDisabled() {
+		return PRECISE_MATH_GLOBALLY_DISABLED;
+	}
```

Comment:
this is a new pattern i really not wanna indroduce? having a getter "isWhateverGlobally" and  "isWhatever" and specially not "Disabled" what inverse the logic